lib/v0_bindings_extensions.rb

```diff
@@ -145,6 +130,38 @@ def summarize(user_uuid:)
     Api::V0::Bindings::GetHighlightsSummaryParameters
   ].each do |klass|
     klass.class_exec do
+
+      def get_highlights(user_id)
+        filters = to_hash.except(:page, :per_page, :sets)
```

Comment:
this seems like a lot of code "parts" that maybe should be written as chainable scopes on the model (some with parameters?)